[2025-02-18T04:31:00.824+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:00.853+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2025-02-18T04:31:00.860+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2025-02-18T04:31:00.860+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:00.876+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-12 00:00:00+00:00
[2025-02-18T04:31:00.881+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=307) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:00.882+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-12T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpmvp_q0fz']
[2025-02-18T04:31:00.884+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask hello_task
[2025-02-18T04:31:00.883+0000] {standard_task_runner.py:72} INFO - Started process 318 to run task
[2025-02-18T04:31:00.907+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:00.937+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:00.939+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:01.026+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-12T00:00:00+00:00'
[2025-02-18T04:31:01.027+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:01.035+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:01.036+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:01.047+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:01.049+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-12T00:00:00+00:00, execution_date=20240312T000000, start_date=20250218T043100, end_date=20250218T043101
[2025-02-18T04:31:01.075+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:01.083+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:31:01.084+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:34:03.902+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:34:03.932+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2025-02-18T16:34:03.941+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2025-02-18T16:34:03.946+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:34:03.984+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-12 00:00:00+00:00
[2025-02-18T16:34:03.994+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1592) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:34:03.995+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-12T00:00:00+00:00', '--job-id', '394', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpc_3kwijk']
[2025-02-18T16:34:03.996+0000] {standard_task_runner.py:72} INFO - Started process 1602 to run task
[2025-02-18T16:34:03.997+0000] {standard_task_runner.py:105} INFO - Job 394: Subtask hello_task
[2025-02-18T16:34:04.058+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:04.121+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-12T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:34:04.122+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:04.231+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-12T00:00:00+00:00'
[2025-02-18T16:34:04.232+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:34:04.246+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:34:04.246+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:34:04.260+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:34:04.264+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-12T00:00:00+00:00, execution_date=20240312T000000, start_date=20250218T163403, end_date=20250218T163404
[2025-02-18T16:34:04.328+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:04.351+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:04.352+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
